NuRadioMC/EvtGen/generator.py

```diff
@@ -895,6 +897,7 @@ def generate_surface_muons(filename, n_events, Emin, Emax,
 
     data_sets["event_ids"] = np.arange(n_events) + start_event_id
     data_sets["n_interaction"] = np.ones(n_events, dtype=np.int)
+    data_sets["vertex_times"] = np.zeros(n_events, dtype=np.int)
```

Comment:
why is this of type int? Shouldn't it be float?

Reply:
Yes.